src/components/common/PrivateRoute.js

```diff
@@ -44,16 +33,15 @@ const PrivateRoute = ({
     allowRoles,
     redirect,
     dispatch,
-    oktaAuth,
-    authState,
     push,
   ]);
+  console.log(profile_id);
```

Comment:
I know this is WIP - Be sure to comment 'why' we just want to log this out for now.